renovate.json

```diff
@@ -42,6 +42,10 @@
     {
       "packageNames": ["javax.servlet:javax.servlet-api"],
       "rangeStrategy": "pin"
+    },
+    {
+      "packageNames": ["com.microsoft.sqlserver:mssql-jdbc"],
```

Comment:
I think a PR with this was already merged?

Reply:
Ah, my branch is stale. I'll rebase it.